### ISSUE TITLE
Remove model description out of autodocs

### ISSUE THREAD

**Issue by hol353** [APSIMInitiative/ApsimX#7100]
Following the @APSIMInitiative/reference-panel meeting this morning we decided to explore the option to:

- remove the model description part out of the auto docs. It was thought that they are too difficult to read and too difficult to maintain. We would retain the PDF autodoc on the web site with just the other sections - validation, sensibility.
- Add a mechanism to the GUI (main menu button?) that quickly lets the user see a model structure and its parameterisation from within the GUI. Perhaps it could open another tab showing the structure for the model you want to look at. This would work independently of what .apsimx file is currently loaded e.g. if you have a wheat simulation loaded you might want to have a quick look at the barley model parameterisation in another tab.
- Ideally @sarchontoulis would like this by April, well before his next training workshop.

**Comment by hol430** [comment]
The second one is already possible, you can right click -> show model structure.

**Comment by sarchontoulis** [comment]
Thanks @hol353 !

**Comment by hol353** [comment]
> The second one is already possible, you can right click -> show model structure.

Yes, but only if you have a simulation loaded with that model in it. We want a button (or something) that shows a list of all models that you can get the model structure for rather than having to find an example and load it.

**Comment by hol430** [comment]
They should all be in the standard toolbox.

**Comment by ric394** [comment]
@hol353 
### Two questions:
- I noticed that the AutoDocumentation class is used to document the usage for PMF nodes and cannot be simply removed without considering if this node documentation is necessary/useful.
   - I may need a rundown on how the auto documentation works before making changes.
- How do the validation and sensitivity docs get generated? Are they generated when a new APSIM version is generated? Are the PMF model docs generated in the same manner?
